main.c

```diff
@@ -69,19 +69,12 @@ int main(int argc, char** argv) {
     // loop timer
     uint32_t last_millis = millis();
 
-    // Set up linear actuator - maybe this shouldn't block forever on failure
-    lin_actuator_init();
+    // Set up valve
+    valve_init();
+
     RED_LED_ON();
-    while (!lin_actuator_dac_init()) {
-        if (millis() - last_millis > MAX_LOOP_TIME_DIFF_ms) {
-            can_msg_t error_msg;
-            build_board_stat_msg(millis(), E_CANNOT_INIT_DACS, NULL, 0, &error_msg);
-            txb_enqueue(&error_msg);
-            last_millis = millis();
-        }
-        txb_heartbeat(); // send out queued CAN messages
-    }
     RED_LED_OFF();
```

Comment:
you turn the red LED on, then off immediately, this is weird and useless, I would just delete both lines for now.

Reply:
Gotcha